contracts/facets/CommitteeFacet.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
```

Comment:
Seems to me the "setFoo" functions should in the paramsfacet, right?

Reply:
Well these which are set by the committee I put here, and the general params set by hats governance I put in the params. Makes sense or what do you think?